### PR TITLE
Add prompt for blockchain settings & add ALD settings

### LINKED ISSUE #1215: Add ALD settings & improve seeds

### PR DESCRIPTION
**Pull request by mederic-p**
Issue/Task Number: #1215
Closes #1215

# Overview

Update plasma settings -> Add support for ALD
Update blockchain seeds -> Add prompts with default value
Hopefully fix flaky test

### CODE REVIEW

**Review by T-Dnzt — APPROVED** [review]
Approving this but expecting the TODOs to be fixed in the deposit PR 👍

File: apps/eth_blockchain/lib/eth_blockchain/childchain.ex [hunk 1]
```diff
@@ -37,6 +37,9 @@ defmodule EthBlockchain.Childchain do
         opts \\ []
       ) do
     with :ok <- check_childchain(childchain_identifier),
```
Comment (kalouo) [inline apps/eth_blockchain/lib/eth_blockchain/childchain.ex:39]: what's the issue in getting rid of this?

File: apps/ewallet/lib/ewallet/gates/transactions/childchain_gate.ex [hunk 2]
```diff
@@ -31,6 +31,7 @@ defmodule EWallet.TransactionGate.Childchain do
   end
 
   defp build_transaction_attrs(%{"address" => address} = attrs) do
+    # this doesn't exist anymore with ALD and need to be replaced with call to the vaults
```
Comment (kalouo) [inline apps/ewallet/lib/ewallet/gates/transactions/childchain_gate.ex:34]: can you replace it in the PR or is this a larger task?

File: apps/ewallet/lib/ewallet/seeders/cli_seeder.ex [hunk 3]
```diff
@@ -135,6 +136,48 @@ defmodule EWallet.Seeder.CLI do
...
+    case byte_size(val) == 0 do
+      true ->
+        {name, process_default(default)}
+
+      false ->
+        {name, Normalize.to_boolean(val)}
+    end
+  rescue
+    Normalize.ToBooleanError ->
+      IO.puts("#{prompt} is not a invalid, please input a valid boolean.")
```
Comment (kalouo) [inline apps/ewallet/lib/ewallet/seeders/cli_seeder.ex]: is not `valid`

Comment (kalouo) [reply apps/ewallet/lib/ewallet/seeders/cli_seeder.ex]: the message below it is correct but I'll let you pick how to make it consistent

File: apps/ewallet/lib/ewallet/gates/transactions/childchain_gate.ex (hunk 2 above)
Comment (mederic-p) [reply apps/ewallet/lib/ewallet/gates/transactions/childchain_gate.ex:34]: it's big :)
It'll be in this PR: https://github.com/omgnetwork/ewallet/pull/1217/files

File: apps/eth_blockchain/lib/eth_blockchain/childchain.ex (hunk 1 above)
Comment (mederic-p) [reply apps/eth_blockchain/lib/eth_blockchain/childchain.ex:39]: All of these things are related to plasma deposits which are revamped to support ALD format in this PR: https://github.com/omgnetwork/ewallet/pull/1217/files

**Review by kalouo — APPROVED** [review]

**Pull request merged**